orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/numeric/GreaterThan.feature

```diff
@@ -389,119 +389,3 @@ Feature: User can specify that a numeric value is higher than, but not equal to,
       | 3.7 |
       | 3.6 |
       | 3.5 |
-
-#after
-  Scenario: greaterThan run against a non contradicting after should be successful (greaterThan 1 AND after 2019-01-01T00:00:00.00)
```

Comment:
I can't believe how many of these tests are testing that constraints of different types don't interfere!